dashboard/app/models/course_offering.rb

```diff
@@ -413,7 +413,7 @@ def csd?
   end
 
   def hoc?
-    category == 'hoc' || marketing_initiative == Curriculum::SharedCourseConstants::COURSE_OFFERING_MARKETING_INITIATIVES.hoc
+    marketing_initiative == Curriculum::SharedCourseConstants::COURSE_OFFERING_MARKETING_INITIATIVES.hoc
```

Comment:
Before making this change, I confirmed that all courses in category `hoc` are already in marketing initiative `HOC`